### ISSUE TITLE
[Platform] Backup failures in itest 2.5.4 b2 on gcp universes

### ISSUE THREAD

**Issue by iSignal** [yugabyte/yugabyte-db#7576]
Backups are failing at the s3cmd put step on this itest. This could be due to the newer centos image we moved to in 5e38f6bcc2b35d042a41df2b9e9cd03d27d32c00

```
2021-03-09 09:40:49,692 ERROR: Failed to run command [[ ssh -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null -i /opt/yugabyte/yugaware/data/keys/f1eb4d19-6d7d-437f-9841-bdb9f82ed197/yb-itest-6ff734e1bf-20210309-091740-key.pem -p 54422 -q yugabyte@10.200.0.38 'cd / &&  bash -c '"'"'s3cmd --force --no-check-certificate --config=/tmp/yb_backup_puabwlgxnijtjvdv/cloud_cfg put /tmp/yb_backup_puabwlgxnijtjvdv/SnapshotInfoPB.sha256 s3://backups.yugabyte.com/test/itest/univ-f345d099-f80a-47ad-bce1-5e29032877ca/backup-2021-03-09T09:40:41-1243304469/table-ybdemo_keyspace.cassandrakeyvalue-116284dd9d394b75ac63d06fbaef43b5/SnapshotInfoPB.sha256'"'"'' ]]: code=72 output=
```

This s3cmd install failure seems to be related to this error at provision time
`'bin/ybcloud.sh gcp --region us-central1 instance provision yb-itest-jenk-i517-9d35484c5c-20210309-091740-n3'`

```
TASK [install_backup_util : s3cmd | Install setuptools] ************************
Tuesday 09 March 2021  09:27:19 +0000 (0:00:04.883)       0:02:41.090 *********
fatal: [10.200.0.37] FAILED! => {"changed": false, "failed": true, "module_stderr": "Shared connection to 10.200.0.37 closed.\r\n", "module_stdout": "Traceback (most recent call last):\r\n  File \"/tmp/ansible_Lmdlom/ansible_module_command.py\", line 212, in <module>\r\n    main()\r\n  File \"/tmp/ansible_Lmdlom/ansible_module_command.py\", line 155, in main\r\n    os.chdir(chdir)\r\nOSError: [Errno 2] No such file or directory: '/usr/setuptools-44.1.1'\r\n", "msg": "MODULE FAILURE"}
...ignoring

TASK [install_backup_util : s3cmd | Install python magic] **********************
Tuesday 09 March 2021  09:27:20 +0000 (0:00:01.210)       0:02:42.300 *********
fatal: [10.200.0.37] FAILED! => {"changed": true, "cmd": ["python", "setup.py", "install"], "delta": "0:00:00.433711", "end": "2021-03-09 09:27:22.044805", "failed": true, "rc": 1, "start": "2021-03-09 09:27:21.611094", "stderr": "Traceback (most recent call last):\n  File \"setup.py\", line 4, in <module>\n    from setuptools import setup\nImportError: No module named setuptools", "stdout": "", "stdout_lines": [], "warnings": []}
...ignoring

TASK [install_backup_util : s3cmd | Build dateutil] ****************************
Tuesday 09 March 2021  09:27:22 +0000 (0:00:01.849)       0:02:44.150 *********
fatal: [10.200.0.37] FAILED! => {"changed": true, "cmd": ["python", "setup.py", "build"], "delta": "0:00:00.440929", "end": "2021-03-09 09:27:23.874886", "failed": true, "rc": 1, "start": "2021-03-09 09:27:23.433957", "stderr": "Traceback (most recent call last):\n  File \"setup.py\", line 5, in <module>\n    import setuptools\nImportError: No module named setuptools", "stdout": "", "stdout_lines": [], "warnings": []}
...ignoring

TASK [install_backup_util : s3cmd | Install dateutil] **************************
Tuesday 09 March 2021  09:27:23 +0000 (0:00:01.774)       0:02:45.924 *********
fatal: [10.200.0.37] FAILED! => {"changed": true, "cmd": ["python", "setup.py", "install"], "delta": "0:00:00.429971", "end": "2021-03-09 09:27:25.570345", "failed": true, "rc": 1, "start": "2021-03-09 09:27:25.140374", "stderr": "Traceback (most recent call last):\n  File \"setup.py\", line 5, in <module>\n    import setuptools\nImportError: No module named setuptools", "stdout": "", "stdout_lines": [], "warnings": []}
...ignoring

TASK [install_backup_util : s3cmd | Install s3cmd] *****************************
Tuesday 09 March 2021  09:27:25 +0000 (0:00:01.701)       0:02:47.625 *********
fatal: [10.200.0.37] FAILED! => {"changed": true, "cmd": ["python", "setup.py", "install"], "delta": "0:00:00.428267", "end": "2021-03-09 09:27:27.406830", "failed": true, "rc": 1, "start": "2021-03-09 09:27:26.978563", "stderr": "Traceback (most recent call last):\n  File \"setup.py\", line 23, in <module>\n    from setuptools import setup\nImportError: No module named setuptools", "stdout": "Using xml.etree.ElementTree for XML processing", "stdout_lines": ["Using xml.etree.ElementTree for XML processing"], "warnings": []}
...ignoring
```

@SergeyPotachev @skorobogatydmitry

**Comment by skorobogatydmitry** [comment]
Issue resolved starting from 2.5.4-b8 itest.

**Comment by streddy-yb** [comment]
Since #7475 is backported to 2.4.2, we need to backport this fix to 2.4.2 as well.

**Comment by SergeyPotachev** [comment]
Backported to 2.4 and to 2.5.3 together with changes from #7475.
